packages/prisma/schema.prisma

```diff
@@ -213,18 +216,20 @@ enum IdentityProvider {
 }
 
 model DestinationCalendar {
-  id           Int         @id @default(autoincrement())
-  integration  String
-  externalId   String
+  id                               Int                   @id @default(autoincrement())
+  integration                      String
+  externalId                       String
   /// @zod.custom(imports.emailSchema)
-  primaryEmail String?
-  user         User?       @relation(fields: [userId], references: [id], onDelete: Cascade)
-  userId       Int?        @unique
-  booking      Booking[]
-  eventType    EventType?  @relation(fields: [eventTypeId], references: [id], onDelete: Cascade)
-  eventTypeId  Int?        @unique
-  credentialId Int?
-  credential   Credential? @relation(fields: [credentialId], references: [id], onDelete: Cascade)
+  primaryEmail                     String?
+  user                             User?                 @relation(fields: [userId], references: [id], onDelete: Cascade)
+  userId                           Int?                  @unique
+  booking                          Booking[]
+  eventType                        EventType?            @relation(fields: [eventTypeId], references: [id], onDelete: Cascade)
+  eventTypeId                      Int?                  @unique
+  credentialId                     Int?
+  credential                       Credential?           @relation(fields: [credentialId], references: [id], onDelete: Cascade)
+  domainWideDelegation             DomainWideDelegation? @relation(fields: [domainWideDelegationCredentialId], references: [id], onDelete: Cascade)
+  domainWideDelegationCredentialId String?
```

Comment:
I don't exactly remember the reason why I added this relation here when we can reach it through credential.delegatedTo.
Maybe it was to avoid the join as it is used during booking and availability check which has to be fast but we should add a reason for this relation in inline code comment

cc @Udit-takkar if you know.

Reply:
> I don't exactly remember the reason why I added this relation here when we can reach it through credential.delegatedTo.

Nah. when you create a workspace platform and domain wide credential no record is created on Credential Table.